Vagrantfile

```diff
@@ -12,20 +12,70 @@ end
 # Vagrantfile API/syntax version. Don't touch unless you know what you're doing!
 VAGRANTFILE_API_VERSION = "2"
 
+# Return a nested hash of amis, indexed by region and root_device_type
+def get_amis
+  require "open-uri"
+  require "json"
+
+  images_uri = "https://dl.flynn.io/ec2/images.json"
+  json = JSON.parse(open(images_uri).read())
+  latest = json["versions"][0]["images"]
+
+  Hash[ latest.map { |ami| [ ami["region"], Hash[ ami["root_device_type"], ami["id"] ] ] } ]
+end
+
 Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
   config.vm.box = "flynn-base"
   config.vm.box_url = "https://dl.flynn.io/vagrant/flynn-base.json"
   config.vm.box_version = "> 0"
 
-  # RFC 5737 TEST-NET-1 used to avoid DNS rebind protection
-  config.vm.network "private_network", ip: "192.0.2.100"
-
-  config.vm.provider "virtualbox" do |v|
+  # VAGRANT_MEMORY          - instance memory, in MB
+  # VAGRANT_CPUS            - instance virtual CPUs
+  config.vm.provider "virtualbox" do |v, override|
     v.memory = ENV["VAGRANT_MEMORY"] || 1024
     v.cpus = ENV["VAGRANT_CPUS"] || 2
+
+    # RFC 5737 TEST-NET-1 used to avoid DNS rebind protection
+    override.vm.network "private_network", ip: "192.0.2.100"
+  end
+
+  # AWS_ACCESS_KEY_ID       - AWS IAM public ID
+  # AWS_SECRET_ACCESS_KEY   - AWS IAM secret key
+  # AWS_SESSION_TOKEN       - AWS IAM role session token
+  # AWS_DEFAULT_REGION      - AWS EC2 region to launch instance in
+  # FLYNN_VAGRANT_KEY       - SSH key (.pem file) used to connect to the instance
+  # FLYNN_VAGRANT_KEYNAME   - AWS EC2 key pair name
+  # FLYNN_VAGRANT_SUBNET    - AWS VPC subnet to launch instance in
+  # FLYNN_VAGRANT_IAM       - AWS IAM role (by name) to give instance permissions
+  # FLYNN_VAGRANT_IAM_ARN   - AWS IAM role (by ARN) to give instance permissions
+  # FLYNN_VAGRANT_TAGS      - AWS EC2 tags to apply to instance (comma separated key-value pairs)
```

Comment:
What do you think about `VAGRANT_AWS_*` or `FLYNN_AWS_*` for the custom AWS-specific variables?